index.rst

```diff
@@ -2,6 +2,613 @@
 
 .. note::
 
-   In progress.
+   ***DRAFT***
+
+   This document outlines each DM communication platform (including
+   chat, ticketing, forums, technical notes and software
+   documentation) and explains what need it fulfils in DM. It also
+   sketches out ways in which these platforms could support future
+   project-wide communication strategies under various policies. 
+
+LSST Data Management uses a number of technical communication and
+documentation platforms to address our needs, with the SQuaRE team
+leading the management (and development, when necessary) of these
+platforms.  Each platform addresses a specific niche of audience (DM,
+Project or our science community), synchronicity (real-time,
+asynchronous, or broadcast), and task (e.g., work ticketing,
+documenting designs, manuals, etc).
+
+In this document we outline SQuaRE's communication support services
+and speculate now they could be used to support a project-wide
+communication strategy. Note, however, that while we have chosen some
+of these platforms for their ability to support a wider outreach into
+the project and the community, they can all be restricted to their
+current audiences if project policy demands it.
+
+DM's communications platform strategy has evolved rapidly since Summer
+2015 with the appointment of our Documentation Engineer, so this
+document also takes the opportunity to foreshadow platform migrations
+that are underway.
+
+How are tools selected
+========================
+
+With all communication tools, there is always a balance of providing
+users a diversity of tools that they want to use and consolidating
+tools so that information does not fragment and to boost the network
+effect. We have found that the parameter space of task, synchronicty
+and audience do require multiple tools, but we favour only having one
+supported tool for each locus in that space.
```

Comment:
Good idea. I'll make slides first, and see if such a chart can emerge from that.